### ISSUE TITLE
1.16 Tweakscale Issues

### ISSUE THREAD

**Issue by Neouni** [sswelm/KSPInterstellar#11]
Some excerpts from the logs:
Related to PFissionReactor

[WRN 21:15:43.457] [TweakScale Warning] No valid member found for PowerOutput in Part
TweakScale.Tools:LogWf(String, Object[])
TweakScale.MemberUpdater:Create(Object, String)
TweakScale.ScaleExponents:UpdateFields(Object, Object, ScalingFactor, Part)
TweakScale.ScaleExponents:UpdateObject(Part, Part, Dictionary`2, ScalingFactor)
TweakScale.TSGenericUpdater:OnRescale(ScalingFactor)
TweakScale.TweakScale:Setup()
TweakScale.TweakScale:OnLoad(ConfigNode)
PartModule:Load(ConfigNode)
Part:LoadModule(ConfigNode, Int32&)
ShipConstruct:LoadShip(ConfigNode)
ShipConstruction:LoadShip(String)
EditorLogic:StartEditor(Boolean)
EditorLogic:shipToLoadSelected(String, String, LoadType)
CraftBrowser:drawList(Int32)
UnityEngine.GUI:CallWindowDelegate(WindowFunction, Int32, GUISkin, Int32, Single, Single, GUIStyle)

[WRN 21:15:43.461] [TweakScale Warning] No valid member found for upgradedPowerOutput in Part
TweakScale.Tools:LogWf(String, Object[])
TweakScale.MemberUpdater:Create(Object, String)
TweakScale.ScaleExponents:UpdateFields(Object, Object, ScalingFactor, Part)
TweakScale.ScaleExponents:UpdateObject(Part, Part, Dictionary`2, ScalingFactor)
TweakScale.TSGenericUpdater:OnRescale(ScalingFactor)
TweakScale.TweakScale:Setup()
TweakScale.TweakScale:OnLoad(ConfigNode)
PartModule:Load(ConfigNode)
Part:LoadModule(ConfigNode, Int32&)
ShipConstruct:LoadShip(ConfigNode)
ShipConstruction:LoadShip(String)
EditorLogic:StartEditor(Boolean)
EditorLogic:shipToLoadSelected(String, String, LoadType)
CraftBrowser:drawList(Int32)
UnityEngine.GUI:CallWindowDelegate(WindowFunction, Int32, GUISkin, Int32, Single, Single, GUIStyle)

[WRN 21:15:43.466] [TweakScale Warning] No valid member found for partMass in Part
TweakScale.Tools:LogWf(String, Object[])
TweakScale.MemberUpdater:Create(Object, String)
TweakScale.ScaleExponents:UpdateFields(Object, Object, ScalingFactor, Part)
TweakScale.ScaleExponents:UpdateObject(Part, Part, Dictionary`2, ScalingFactor)
TweakScale.TSGenericUpdater:OnRescale(ScalingFactor)
TweakScale.TweakScale:Setup()
TweakScale.TweakScale:OnLoad(ConfigNode)
PartModule:Load(ConfigNode)
Part:LoadModule(ConfigNode, Int32&)
ShipConstruct:LoadShip(ConfigNode)
ShipConstruction:LoadShip(String)
EditorLogic:StartEditor(Boolean)
EditorLogic:shipToLoadSelected(String, String, LoadType)
CraftBrowser:drawList(Int32)
UnityEngine.GUI:CallWindowDelegate(WindowFunction, Int32, GUISkin, Int32, Single, Single, GUIStyle)

[WRN 21:15:43.470] [TweakScale Warning] No valid member found for radius in Part
TweakScale.Tools:LogWf(String, Object[])
TweakScale.MemberUpdater:Create(Object, String)
TweakScale.ScaleExponents:UpdateFields(Object, Object, ScalingFactor, Part)
TweakScale.ScaleExponents:UpdateObject(Part, Part, Dictionary`2, ScalingFactor)
TweakScale.TSGenericUpdater:OnRescale(ScalingFactor)
TweakScale.TweakScale:Setup()
TweakScale.TweakScale:OnLoad(ConfigNode)
PartModule:Load(ConfigNode)
Part:LoadModule(ConfigNode, Int32&)
ShipConstruct:LoadShip(ConfigNode)
ShipConstruction:LoadShip(String)
EditorLogic:StartEditor(Boolean)
EditorLogic:shipToLoadSelected(String, String, LoadType)
CraftBrowser:drawList(Int32)
UnityEngine.GUI:CallWindowDelegate(WindowFunction, Int32, GUISkin, Int32, Single, Single, GUIStyle)

[WRN 21:15:43.602] [TweakScale Warning] No valid member found for PowerOutput in Part
TweakScale.Tools:LogWf(String, Object[])
TweakScale.MemberUpdater:Create(Object, String)
TweakScale.ScaleExponents:UpdateFields(Object, Object, ScalingFactor, Part)
TweakScale.ScaleExponents:UpdateObject(Part, Part, Dictionary`2, ScalingFactor)
TweakScale.TSGenericUpdater:OnRescale(ScalingFactor)
TweakScale.TweakScale:Setup()
TweakScale.TweakScale:OnLoad(ConfigNode)
PartModule:Load(ConfigNode)
Part:LoadModule(ConfigNode, Int32&)
ShipConstruct:LoadShip(ConfigNode)
ShipConstruction:LoadShip(String)
EditorLogic:StartEditor(Boolean)
EditorLogic:shipToLoadSelected(String, String, LoadType)
CraftBrowser:drawList(Int32)
UnityEngine.GUI:CallWindowDelegate(WindowFunction, Int32, GUISkin, Int32, Single, Single, GUIStyle)

[WRN 21:15:43.607] [TweakScale Warning] No valid member found for upgradedPowerOutput in Part
TweakScale.Tools:LogWf(String, Object[])
TweakScale.MemberUpdater:Create(Object, String)
TweakScale.ScaleExponents:UpdateFields(Object, Object, ScalingFactor, Part)
TweakScale.ScaleExponents:UpdateObject(Part, Part, Dictionary`2, ScalingFactor)
TweakScale.TSGenericUpdater:OnRescale(ScalingFactor)
TweakScale.TweakScale:Setup()
TweakScale.TweakScale:OnLoad(ConfigNode)
PartModule:Load(ConfigNode)
Part:LoadModule(ConfigNode, Int32&)
ShipConstruct:LoadShip(ConfigNode)
ShipConstruction:LoadShip(String)
EditorLogic:StartEditor(Boolean)
EditorLogic:shipToLoadSelected(String, String, LoadType)
CraftBrowser:drawList(Int32)
UnityEngine.GUI:CallWindowDelegate(WindowFunction, Int32, GUISkin, Int32, Single, Single, GUIStyle)

[WRN 21:15:43.612] [TweakScale Warning] No valid member found for partMass in Part
TweakScale.Tools:LogWf(String, Object[])
TweakScale.MemberUpdater:Create(Object, String)
TweakScale.ScaleExponents:UpdateFields(Object, Object, ScalingFactor, Part)
TweakScale.ScaleExponents:UpdateObject(Part, Part, Dictionary`2, ScalingFactor)
TweakScale.TSGenericUpdater:OnRescale(ScalingFactor)
TweakScale.TweakScale:Setup()
TweakScale.TweakScale:OnLoad(ConfigNode)
PartModule:Load(ConfigNode)
Part:LoadModule(ConfigNode, Int32&)
ShipConstruct:LoadShip(ConfigNode)
ShipConstruction:LoadShip(String)
EditorLogic:StartEditor(Boolean)
EditorLogic:shipToLoadSelected(String, String, LoadType)
CraftBrowser:drawList(Int32)
UnityEngine.GUI:CallWindowDelegate(WindowFunction, Int32, GUISkin, Int32, Single, Single, GUIStyle)

[WRN 21:15:43.616] [TweakScale Warning] No valid member found for radius in Part
TweakScale.Tools:LogWf(String, Object[])
TweakScale.MemberUpdater:Create(Object, String)
TweakScale.ScaleExponents:UpdateFields(Object, Object, ScalingFactor, Part)
TweakScale.ScaleExponents:UpdateObject(Part, Part, Dictionary`2, ScalingFactor)
TweakScale.TSGenericUpdater:OnRescale(ScalingFactor)
TweakScale.TweakScale:Setup()
TweakScale.TweakScale:OnLoad(ConfigNode)
PartModule:Load(ConfigNode)
Part:LoadModule(ConfigNode, Int32&)
ShipConstruct:LoadShip(ConfigNode)
ShipConstruction:LoadShip(String)
EditorLogic:StartEditor(Boolean)
EditorLogic:shipToLoadSelected(String, String, LoadType)
CraftBrowser:drawList(Int32)
UnityEngine.GUI:CallWindowDelegate(WindowFunction, Int32, GUISkin, Int32, Single, Single, GUIStyle)

[WRN 21:15:43.621] [TweakScale Warning] No valid member found for partMass in FNGenerator
TweakScale.Tools:LogWf(String, Object[])
TweakScale.MemberUpdater:Create(Object, String)
TweakScale.ScaleExponents:UpdateFields(Object, Object, ScalingFactor, Part)
TweakScale.ScaleExponents:UpdateObject(Part, Part, Dictionary`2, ScalingFactor)
TweakScale.TSGenericUpdater:OnRescale(ScalingFactor)
TweakScale.TweakScale:Setup()
TweakScale.TweakScale:OnLoad(ConfigNode)
PartModule:Load(ConfigNode)
Part:LoadModule(ConfigNode, Int32&)
ShipConstruct:LoadShip(ConfigNode)
ShipConstruction:LoadShip(String)
EditorLogic:StartEditor(Boolean)
EditorLogic:shipToLoadSelected(String, String, LoadType)
CraftBrowser:drawList(Int32)
UnityEngine.GUI:CallWindowDelegate(WindowFunction, Int32, GUISkin, Int32, Single, Single, GUIStyle)

[WRN 21:15:43.951] [TweakScale Warning] No valid member found for PowerOutput in Part
TweakScale.Tools:LogWf(String, Object[])
TweakScale.MemberUpdater:Create(Object, String)
TweakScale.ScaleExponents:UpdateFields(Object, Object, ScalingFactor, Part)
TweakScale.ScaleExponents:UpdateObject(Part, Part, Dictionary`2, ScalingFactor)
TweakScale.TSGenericUpdater:OnRescale(ScalingFactor)
TweakScale.TweakScale:Setup()
TweakScale.TweakScale:OnLoad(ConfigNode)
PartModule:Load(ConfigNode)
Part:LoadModule(ConfigNode, Int32&)
ShipConstruct:LoadShip(ConfigNode)
ShipConstruction:LoadShip(String)
EditorLogic:StartEditor(Boolean)
EditorLogic:shipToLoadSelected(String, String, LoadType)
CraftBrowser:drawList(Int32)
UnityEngine.GUI:CallWindowDelegate(WindowFunction, Int32, GUISkin, Int32, Single, Single, GUIStyle)

[WRN 21:15:43.956] [TweakScale Warning] No valid member found for upgradedPowerOutput in Part
TweakScale.Tools:LogWf(String, Object[])
TweakScale.MemberUpdater:Create(Object, String)
TweakScale.ScaleExponents:UpdateFields(Object, Object, ScalingFactor, Part)
TweakScale.ScaleExponents:UpdateObject(Part, Part, Dictionary`2, ScalingFactor)
TweakScale.TSGenericUpdater:OnRescale(ScalingFactor)
TweakScale.TweakScale:Setup()
TweakScale.TweakScale:OnLoad(ConfigNode)
PartModule:Load(ConfigNode)
Part:LoadModule(ConfigNode, Int32&)
ShipConstruct:LoadShip(ConfigNode)
ShipConstruction:LoadShip(String)
EditorLogic:StartEditor(Boolean)
EditorLogic:shipToLoadSelected(String, String, LoadType)
CraftBrowser:drawList(Int32)
UnityEngine.GUI:CallWindowDelegate(WindowFunction, Int32, GUISkin, Int32, Single, Single, GUIStyle)

[WRN 21:15:43.961] [TweakScale Warning] No valid member found for partMass in Part
TweakScale.Tools:LogWf(String, Object[])
TweakScale.MemberUpdater:Create(Object, String)
TweakScale.ScaleExponents:UpdateFields(Object, Object, ScalingFactor, Part)
TweakScale.ScaleExponents:UpdateObject(Part, Part, Dictionary`2, ScalingFactor)
TweakScale.TSGenericUpdater:OnRescale(ScalingFactor)
TweakScale.TweakScale:Setup()
TweakScale.TweakScale:OnLoad(ConfigNode)
PartModule:Load(ConfigNode)
Part:LoadModule(ConfigNode, Int32&)
ShipConstruct:LoadShip(ConfigNode)
ShipConstruction:LoadShip(String)
EditorLogic:StartEditor(Boolean)
EditorLogic:shipToLoadSelected(String, String, LoadType)
CraftBrowser:drawList(Int32)
UnityEngine.GUI:CallWindowDelegate(WindowFunction, Int32, GUISkin, Int32, Single, Single, GUIStyle)

[WRN 21:15:43.965] [TweakScale Warning] No valid member found for radius in Part
TweakScale.Tools:LogWf(String, Object[])
TweakScale.MemberUpdater:Create(Object, String)
TweakScale.ScaleExponents:UpdateFields(Object, Object, ScalingFactor, Part)
TweakScale.ScaleExponents:UpdateObject(Part, Part, Dictionary`2, ScalingFactor)
TweakScale.TSGenericUpdater:OnRescale(ScalingFactor)
TweakScale.TweakScale:Setup()
TweakScale.TweakScale:OnLoad(ConfigNode)
PartModule:Load(ConfigNode)
Part:LoadModule(ConfigNode, Int32&)
ShipConstruct:LoadShip(ConfigNode)
ShipConstruction:LoadShip(String)
EditorLogic:StartEditor(Boolean)
EditorLogic:shipToLoadSelected(String, String, LoadType)
CraftBrowser:drawList(Int32)
UnityEngine.GUI:CallWindowDelegate(WindowFunction, Int32, GUISkin, Int32, Single, Single, GUIStyle)

[WRN 21:15:43.970] [TweakScale Warning] No valid member found for powerRequirements in InterstellarFissionPBDP
TweakScale.Tools:LogWf(String, Object[])
TweakScale.MemberUpdater:Create(Object, String)
TweakScale.ScaleExponents:UpdateFields(Object, Object, ScalingFactor, Part)
TweakScale.ScaleExponents:UpdateObject(Part, Part, Dictionary`2, ScalingFactor)
TweakScale.TSGenericUpdater:OnRescale(ScalingFactor)
TweakScale.TweakScale:Setup()
TweakScale.TweakScale:OnLoad(ConfigNode)
PartModule:Load(ConfigNode)
Part:LoadModule(ConfigNode, Int32&)
ShipConstruct:LoadShip(ConfigNode)
ShipConstruction:LoadShip(String)
EditorLogic:StartEditor(Boolean)
EditorLogic:shipToLoadSelected(String, String, LoadType)
CraftBrowser:drawList(Int32)
UnityEngine.GUI:CallWindowDelegate(WindowFunction, Int32, GUISkin, Int32, Single, Single, GUIStyle)

[WRN 21:15:44.176] [TweakScale Warning] Failed to convert string value "0.625 1.25, 1.875, 2.5" to type Single
TweakScale.Tools:LogWf(String, Object[])
TweakScale.Tools:ConvertString(String, Single[])
TweakScale.Tools:ConfigValue(ConfigNode, String, Single[])
TweakScale.ScaleType:.ctor(ConfigNode)
TweakScale.TweakScale:Setup()
TweakScale.TweakScale:OnLoad(ConfigNode)
PartModule:Load(ConfigNode)
Part:LoadModule(ConfigNode, Int32&)
ShipConstruct:LoadShip(ConfigNode)
ShipConstruction:LoadShip(String)
EditorLogic:StartEditor(Boolean)
EditorLogic:shipToLoadSelected(String, String, LoadType)
CraftBrowser:drawList(Int32)
UnityEngine.GUI:CallWindowDelegate(WindowFunction, Int32, GUISkin, Int32, Single, Single, GUIStyle)

[WRN 21:15:44.292] [TweakScale Warning] Failed to convert string value "0.625 1.25, 1.875, 2.5" to type Single
TweakScale.Tools:LogWf(String, Object[])
TweakScale.Tools:ConvertString(String, Single[])
TweakScale.Tools:ConfigValue(ConfigNode, String, Single[])
TweakScale.ScaleType:.ctor(ConfigNode)
TweakScale.TweakScale:Setup()
TweakScale.TweakScale:OnLoad(ConfigNode)
PartModule:Load(ConfigNode)
Part:LoadModule(ConfigNode, Int32&)
ShipConstruct:LoadShip(ConfigNode)
ShipConstruction:LoadShip(String)
EditorLogic:StartEditor(Boolean)
EditorLogic:shipToLoadSelected(String, String, LoadType)
CraftBrowser:drawList(Int32)
UnityEngine.GUI:CallWindowDelegate(WindowFunction, Int32, GUISkin, Int32, Single, Single, GUIStyle)

**Comment by ryanbmilbourne** [comment]
I'm seeing similar warnings running 1.3.5 on KSPv1.0.4 with TweakScalev2.2.1:

```
[Warning]: [TweakScale Warning] No valid member found for powerRequirements in InterstellarFissionPBDP
TweakScale.Tools:LogWf(String, Object[])
TweakScale.MemberUpdater:Create(Object, String)
TweakScale.ScaleExponents:UpdateFields(Object, Object, ScalingFactor, Part)
TweakScale.ScaleExponents:UpdateObject(Part, Part, Dictionary`2, ScalingFactor)
TweakScale.TSGenericUpdater:OnRescale(ScalingFactor)
TweakScale.TweakScale:Setup()
TweakScale.TweakScale:OnLoad(ConfigNode)
PartModule:Load(ConfigNode)
Part:LoadModule(ConfigNode, Int32&)
ShipConstruct:LoadShip(ConfigNode)
ShipConstruction:LoadShip(String)
EditorLogic:StartEditor(Boolean)
EditorLogic:shipToLoadSelected(String, String, LoadType)
CraftBrowser:drawList(Int32)
UnityEngine.GUI:CallWindowDelegate(WindowFunction, Int32, GUISkin, Int32, Single, Single, GUIStyle)
```

**Comment by sswelm** [comment]
Alright, I remove powerRequirements

**Comment by ryanbmilbourne** [comment]
Looks to be resolved in v1.3.6! You can probably close this issue. :+1:

**Comment by sswelm** [comment]
Thanks for your confirmation
